### DISCUSSION TITLE
Not all asset fields are showing up

### DISCUSSION THREAD

**Discussion by keithmancuso** [engram-design/FeedMe#885] [Ideas]
### Description
Not sure if this is a bug or a feature enhancement but it looks like not all the fields on an asset are showing up for import. I have a plaintext field as well as an entries field and only the plaintext field is allowing me to import it

I swear this was working for me before at one point?

### Steps to reproduce

1. Add a plaintext field and an entries fields onto a volume
2. Try and import assets field from a json feed

### Additional info

- Plugin version: 3.0.0-beta.21
- Craft version:Craft Pro 3.0.26.1

**Comment by engram-design** [comment]
We did pare this back (silently) due to imports getting incredibly complicated and some instability (think Matrix + Super Table + Asset field + Entry field), so we've just allowed simple fields to have their data mapped in Element Fields.

We'll bring this back eventually after some further testing.
